tests/e2e/reflect_helper.go

```diff
@@ -16,9 +18,18 @@ import (
 	"github.com/CosmWasm/wasmd/x/wasm/types"
 )
 
-// InstantiateReflectContract store and instantiate a reflect contract instance
+// InstantiateStargateReflectContract stores and instantiates a pre 2.0 reflect contract instance.
+// This instance still expects the old CosmosMsg.Stargate variant instead of the new CosmosMsg.Any.
+func InstantiateStargateReflectContract(t *testing.T, chain *ibctesting.TestChain) sdk.AccAddress {
+	codeID := chain.StoreCodeFile("../../x/wasm/keeper/testdata/stargate_reflect.wasm").CodeID
```

Comment:
Woudn't it be easier to manage if we used the file name `reflect_1_4.wasm` here (and download file from https://github.com/CosmWasm/cosmwasm/releases/tag/v1.4.3). the file itself should not care too much why it is used here in the tests. No strong opinion on 1.2, 1.3, 1.4, or 1.5 – just something old.